apps/keyboard/js/layouts/emoji.js

```diff
@@ -403,12 +475,104 @@ Keyboards.emoji = {
       ]
     ]
     }, {     // Page 4 - Symbols
-    panelKeys: [ { compositeKey: '🆗', type: 'emoji' }, { compositeKey: '🔀', type: 'emoji' },
-                 { compositeKey: '🔁', type: 'emoji' }, { compositeKey: '🔂', type: 'emoji' },
-                 { compositeKey: '🆕', type: 'emoji' }, { compositeKey: '🆙', type: 'emoji' },
-                 { compositeKey: '🆒', type: 'emoji' }, { compositeKey: '🆓', type: 'emoji' },
-                 { compositeKey: '🆖', type: 'emoji' }, { compositeKey: '📶', type: 'emoji' },
-                 { compositeKey: '🎦', type: 'emoji' }, { compositeKey: '🈁', type: 'emoji' }
+    panelKeys: [{ compositeKey: '3', type: 'emoji' }, { compositeKey: '0', type: 'emoji' },
```

Comment:
Are you trying to target 1⃣, which is U+0031 U+20E3 ? If so these keys should be defined as such and the font should encode the glyph as such.

Neither is correctly done currently.